Sources/SwiftTemplate.swift

```diff
@@ -3,9 +3,16 @@
 // Copyright (c) 2017 SwiftGen
 // MIT Licence
 //
-
+import Foundation
 import Stencil
 
+#if os(Linux)
+#if swift(>=3.1)
```

Comment:
I think its important because dependency bindings between podspec and spm are different, you might end up in situation when we update podspec dependency and forget SPM one and swift test / build would prevent it